kannon/master.py

```diff
@@ -10,37 +11,31 @@
 from .task import TaskOnBullet
 from .kube_util import create_job, JobStatus, gen_job_name, get_job_status
 
-
 logger = logging.getLogger(__name__)
 
 
 class Kannon:
+
     def __init__(
         self,
+        # k8s resources
         api_instance: client.BatchV1Api,
-        namespace: str,
-        image_name: str,
-        container_name: str,
-        service_account_name: str,
+        template_job: client.V1Job,
+        # kannon resources
         job_prefix: str,
-        path_child_script: str,
-        env_to_inherit: List[str],
-        backoff_limit: int = 0,
+        path_child_script: str = "./run_child.py",
+        env_to_inherit: List[str] = ["TASK_WORKSPACE_DIRECTORY"],
```

Comment:
in python, you should(must) not use mutable variable for default arguments.

changes for those variables will affect to next call, means another instance creation 😨 
see: 
* https://docs.python.org/3/faq/programming.html#why-are-default-values-shared-between-objects
* https://blog.cosnomi.com/posts/1471/

instead, 

```python
def __init__(
    ...
    env_to_inherit: List[str] | None = None,
    ...
):
    ...
    if env_to_inherit is None:
        env_to_inherit = ["TASK_WORKSPACE_DIRECTORY"]
```

Reply:
Thanks!
Supported in 88d71ff